src/tests/class_tests/openms/source/Matrix_test.cpp

```diff
@@ -54,47 +54,31 @@ Matrix<int> mi;
 
 START_SECTION((void resize(size_type i, size_type j, value_type value = value_type())))
 {
-	mi.resize(2,2,3);
+  mi.resize(2,2,3);
   STATUS("mi1:\n"<< mi);
-	mi.resize(2,3,7);
+  mi.resize(2,3,7);
```

Comment:
I changed the logic here... resizing with fill value always fills the full matrix... 
Note to myself: maybe that is the reason the old TMT code was not the identity matrix - even though it should be?